packages/@coorpacademy-components/src/atom/input-textarea/style.css

```diff
@@ -231,17 +232,17 @@
 }
 
 .coorpmanager textarea {
-  width: 285px;
+  width: 99%;
```

Comment:
to avoid this 
![image](https://user-images.githubusercontent.com/58167920/142178507-d8dfde21-7fe6-42d1-8d5e-f0fec9af00d8.png)

Reply:
@DjamelSoualmi je regarde ça

Reply:
ce style a un padding-right puis un padding... @esa-coorp tout ça ne s'annule pas ? Est-ce que ça vient pas de ça le fait qu'on puisse pas mettre le 100% ?